src/reducers/SearchSlice.jsx

```diff
@@ -34,6 +47,18 @@ const search = createSlice({
     updateRepository(state, action) {
       state.repository = action.payload;
     },
+    updateDropdownError(state, action) {
+      state.dropdownError = action.payload;
+    },
+    updateDropdownHelperText(state, action) {
+      state.dropdownHelperText = action.payload;
+    },
+    updateInputError(state, action) {
+      state.inputError = action.payload;
+    },
+    updateInputHelperText(state, action) {
+      state.inputHelperText = action.payload;
+    },
```

Comment:
It's not 100% clear to me, what the difference between inputError and dropdownError are, it might be a good idea to add comments to explain this here.